### PR TITLE
PR-005_CSS & Home & Member

### PR DESCRIPTION
**Pull request by joelmayemba**
+ homepage update ans css
+ Global css uupdate
+ Members page update

### CODE REVIEW

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/View/Home/index.html.twig [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+	<link rel="stylesheet" href="assets/css/banner.css"/>
+{% endblock %}
+{% block header %}{% endblock %}
+{% block content %}
+	<div id="carouselExampleAutoplaying" class="carousel slide" data-bs-ride="carousel">
+		<div class="main-banner">
+			<div class="carousel-inner">
+				<div class="carousel-item active">
+					<div class="item item-1">
+						<div class="header-text">
```
[the remark this answers is not in the document]
Comment (joelmayemba) [reply src/View/Home/index.html.twig:16]: done

**Review by chouaibj — APPROVED** [review]

File: src/View/Home/index.html.twig
[the remark this answers is not in the document]
Comment (joelmayemba) [reply src/View/Home/index.html.twig:1]: Done

File: src/View/Members/index.html.twig
[the remark this answers is not in the document]
Comment (joelmayemba) [reply src/View/Members/index.html.twig:1]: Done

File: src/View/Home/index.html.twig [hunk 2]
```diff
@@ -12,32 +12,32 @@
...
-						<div class="header-text">
-							<a href="mocktails">
-								<span class="category">Cocktails</a>
-							</span>
-							<h2>
-								Venez!<br>vite découvrir nos mocktails</h2>
+					
+						<div class="item item-1">
+							<div class="header-text">
+								{# <span class="category">Cocktails
```
Comment (Benja-san) [inline src/View/Home/index.html.twig:18]: remove dead code

**Review by Benja-san — APPROVED** [review]

**Pull request merged**